libraries/Microsoft.Bot.Builder.Azure/AppInsightsLoggerMiddleware.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Bot.Schema;
+
+[assembly: InternalsVisibleTo("Microsoft.Bot.Builder.Azure.Tests")]
+
+namespace Microsoft.Bot.Builder.Azure
+{
+    /// <summary>
+    /// Middleware for logging incoming activitites into Application Insights.
+    /// In addition, registers a service that other Application Insights components can log
+    /// telemetry.
+    /// If this component is not registered, visibility within the Bot is not logged.
+    /// </summary>
+    public class AppInsightsLoggerMiddleware : IMiddleware
+    {
+        public static readonly string AppInsightsServiceKey = "${nameof(AppInsightsLoggerMiddleware).AppInsightsContext}";
+        public static readonly string BotMsgEvent = "BotMessageReceived";
+        private static TelemetryClient _telemetryClient;
+        private static bool _logUserName;
+        private static bool _logOriginalMessage;
```

Comment:
I'm still showing these as static variables. Did a commit / merge get missed?

Reply:
Doh, properties in wrong branch.